m4_sessions/1901-inning/prejudice.md

```diff
@@ -1,30 +1,37 @@
 ## Group Members:
-
-
+Tanjie, Melvin, Derek, Harper
 
 ## Name of Session: 
+Prejudice
 
 ## Session Description 
-
 Turing's mission is to unlock human potential by training a diverse, inclusive student body to succeed in high-fulfillment technical careers. In this session we will identify, examine, and discuss our prejudices, biases, and our related experiences with the aim of creating concrete actionable solution to make tech and our broader communities more (inclusive**?)
 
 ## Session Outcomes 
-
 (What will students learn/leave with?): 
+Students will leave with a concrete plan for address their own biases and those of others in the wild
+
+## Session Outline
+* Intros (5-10 mins) Let students settle in the space, an icebreaker where each student writes an interesting fact about themselves on a piece of paper and puts it in a hat. Papers are drawn by a facilitator, the fact is read aloud and the group tries to guess who submitted the fact.  Norms and expectations will then be outlined. 
+
+* Implicit Bias Tests (10 - 15 mins) Students will need to bring a computer to this session.  Facilitators will choose four implicit bias tests.  Students will be randomly assigned to take one of the four test, and time permitting, will be able to another bias test of their choosing. Students will be given a few minutes to reflect on the finding either in a journal or just to themselves.
+
+* Small Group Discussion (15 mins) Students will break into small groups determined by the randomly assigned bias test (eg, those who took the bias around disabilities test will group together, those who took the racial biases test will group, etc.
+The facilitators will write discussion questions on the white board in order to lead focused discussion. All facilitators will have taken a bias test beforehand, and will be prepared to speak aobut their experiences while taking it with their groups.
+
+* Recap and Segue (5 min) Come back together to a larger group, where facilitators will invite students to share insights from their break out discussions.  Facilitator will then provide a link to short article on bias in technology (Article currently TBD, but will center around bias in AI, eg, the racist twitter bot).
+
+* Time allotment for reading the article (5 min)
+
+* Discussion (20 mins) led by all 3 group members.  Discuss how this related to tech.  Start with broad example like racist twitterbot and then bring it down to a more personal level with examples from Turing.  Ask what are steps to take when things like this happen in your community.  Brainstorm potential reactions/actions to be better prepared to confont bias when it presents itself. 
```

Comment:
Think about having some good questions here or even...what if people say x, y, z...how do you respond?